cypress/e2e/topics/shareAndImportTopicInDifferentSchoolOnStaging.feature

```diff
@@ -0,0 +1,65 @@
+@regression_test
+@stable_test
+@group_V
+
+# Note: This feature can not be currently executed using the school API, as creating two different school within the same scenario is not possible. And creating them in two different scenarios results in separate sessions, which prevents the copied board URL from the first scenario from being used in another scenario.
+
+Feature: Topic - Teacher shares a topic to other teacher from different school and the other teacher from different school can import it into their course
```

Comment:
- `Topics` not `Topic`
- also shorten the sentence otherwise it will break report formatting